core/src/main/java/org/apache/iceberg/avro/Avro.java

```diff
@@ -82,11 +96,11 @@ public static WriteBuilder write(OutputFile file) {
 
   public static class WriteBuilder {
     private final OutputFile file;
+    private final Map<String, String> config = Maps.newHashMap();
```

Comment:
Q: I saw that the key-values from`config` are not actually used except the `key = write.avro.compression-codec`, is this correct ?

Reply:
That's correct right now. We have been gradually expanding the settings that are supported in Parquet and I expect that to happen here, too. For example, when we add Zstd support for Avro, we will probably also want to support the compression level setting. Parquet also copies config into the Hadoop `Configuration`, which we could do for Avro if there is a use case.